composer.json

```diff
@@ -16,23 +16,22 @@
     ],
     "require": {
         "composer/installers": "^1.9",
-        "drupal/admin_toolbar": "^3.2",
+        "drupal/admin_toolbar": "^3.3",
         "drupal/autotitle": "^1.2",
-        "drupal/core-composer-scaffold": "^9.4",
-        "drupal/core-project-message": "^9.4",
-        "drupal/core-recommended": "^9.4",
+        "drupal/core-composer-scaffold": "9.5.3",
+        "drupal/core-project-message": "9.5.3",
+        "drupal/core-recommended": "9.5.3",
         "drupal/diff": "^1.1",
         "drupal/feeds": "^3.0@beta",
-        "drupal/geofield": "^1.46",
-        "drupal/gin": "^3",
-        "drupal/gin_login": "^1.3",
+        "drupal/geofield": "^1.52",
+        "drupal/gin": "^3.0@RC",
```

Comment:
Let's try to have the modules compatible with major instead of minor unless we have a really good reason not to.
"^3"

Reply:
done